libs/gamify/data/prisma/schema.prisma

```diff
@@ -26,4 +26,17 @@ model Application {
     externalApplicationUrl String?
     ownerUser              User     @relation(fields: [ownerUserId], references: [id])
     ownerUserId            Int
+    Badge                  Badge[]
+}
+
+model Badge {
+    id                   Int         @id @default(autoincrement())
+    createdAt            DateTime    @default(now())
+    name                 String
+    application          Application @relation(fields: [applicationId], references: [id])
+    applicationId        Int
+    iconPath             String?
+    tier                 String
+    repeatedlyObtainable Boolean
+    totalScore           Int?
```

Comment:
I don't fully understand the purpose of `totalScore` yet.

Reply:
That is with the goal in mind to keep progress of how far the user is to obtaining a badge